### ISSUE TITLE
[Bug] pot crash with numpy binary incompatibility

### ISSUE THREAD

**Issue by tarepan** [openvinotoolkit/openvino#10938]
##### System information (version)
- OpenVINO => 2022.1.0.dev20220302 through pypi
- Operating System / Platform => Ubuntu 18.04.5 LTS (Google Colaboratory)

##### Detailed description
During openvino-dev install through pip, it automatically downgrade numpy version.  
After installed, `pot -h` CLI crash with numpy binary incompatibility error.  
If we manually re-upgrade numpy then re-run `pot -h`, it work fine.  
It suggests that `openvino-dev` has wrong numpy dependency.  

##### Steps to reproduce
[***Reproducing Notebook with the error***](https://colab.research.google.com/drive/1w7FZyw0fbI4o9s79Kn45WBDBflDQezhn?usp=sharing)  

In short, we can reproduce the bug.  
```bash
!pip install "openvino-dev===2022.1.0.dev20220302"
!pot -h

Traceback (most recent call last):
...
  File "pycocotools/_mask.pyx", line 1, in init pycocotools._mask
ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 88 from C header, got 80 from PyObject
```

##### Issue submission checklist

 - [x] I report the issue, it's not a question
 - [x] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
 - [x] There is reproducer code and related data files: images, videos, models, etc.

**Comment by jgespino** [comment]
Hi @tarepan 

Thanks for reporting and providing the steps to reproduce. I was able to reproduce the issue on Google Colab, however, I was not able to reproduce on my local system running Ubuntu 18.04 with numpy 1.19.5. I will need to check with the development team if OpenVINO has been validated on Google Colab and if this is a bug. I will let you know what I find out. 

Regards,
Jesus

**Comment by AlexKoff88** [comment]
In the logs, I can see that there is some incompatibility between the pycocotools dependency of AccuracyChecker and the numpy version. @eaidova, can you please take a look?

**Comment by eaidova** [comment]
@AlexKoff88 @tarepan  we already have internal discussion for this problem. Unfortunately, until openvino does not relax numpy restriction from 1.19 we can not fix this issue... The workaround is to reinstall pycocotools after openvino-dev package installation in the colab environment

Colab environment has preinstalled numpy 1.21 and pycocotools. This package has C++ compiled part and use numpy for its building, so changing numpy version after its installation leads to binary incompatibility

**Comment by AlexKoff88** [comment]
Thanks, Ekaterina! I think this answers the question of why it is reproduced in Collab only.

**Comment by jgespino** [comment]
Closing, please re-open if additional assistance is needed.
